R/Sound_DMR.R

```diff
@@ -575,12 +575,14 @@ run_model <- function(data, i, Output_Frame, formula, model_type,
                       individual_name_z = ''){
   if (model_type == 'binomial') {
     tryCatch({
-      Output_Frame <- run_binomial(data, i, Output_Frame, formula, 'bobyqa')
+      ith_model_summary <- run_binomial(data, i, Output_Frame, formula, 'bobyqa')
```

Comment:
Is Output frame still an argument? Didn't you remove it above?

Reply:
good catch! might have forgotten to remove that while updating..

Reply:
Given that this package is now publicly available, it would be a good idea to start adding some tests so that such changes don't cause anyone inadvertent damage.  Seems like no one is actually going to use it immediately but definitely something to consider.